python/lib/bidsreader.py

```diff
@@ -1,13 +1,12 @@
 """Reads a BIDS structure into a data dictionary using bids.grabbids."""
 
 import re
-import os
 import sys
 import json
 
 import lib.exitcode
 import lib.utilities as utilities
-import bids
+# import bids
```

Comment:
shouldn't this be removed instead of commented out? Or at list moved in the `load_bids_data` function?